inc/alert.class.php

```diff
@@ -82,22 +87,27 @@ function clear($itemtype, $ID, $alert_type) {
     *
     * @since version 0.84
     *
-    * @param $itemtype   ID of the type to clear
-    * @param $ID         ID of the item to clear
+    * @param string  $itemtype ID of the type to clear
+    * @param integer $ID       ID of the item to clear
     *
     * @return boolean
-   **/
+    */
    function cleanDBonItemDelete($itemtype, $ID) {
 
       return $this->deleteByCriteria(['itemtype' => $itemtype, 'items_id' => $ID], 1);
    }
 
 
    /**
-    * @param $options array
-   **/
+    *
```

Comment:
Missing description (I guess "Build a dropdown" or "Show a dropdonw" - I've not checkd in the code)